src/ibattle.h

```diff
@@ -84,7 +84,8 @@ struct BattleOptions
 {
 	BattleOptions()
 	    : battleid(-1)
-	    , islocked(false)
+	    , channel_name("")
+		, islocked(false)
```

Comment:
invalid ident

Reply:
there seem to be many cases where tabs/ws are mixed - often tabs are mostly used and then ws is used for lists of vars; this seems bad style so where you've flagged these i'll "fix" by converting ws to tabs